doc/examples/segmentation/plot_quickshift_full_tree.py

```diff
@@ -0,0 +1,161 @@
+"""
+=============================================================
+Comparison of quickshift with and without full_search enabled
+=============================================================
+
+This example compares the computation of a complete superpixel hierarchy
+with the default which is to limit the parent to a local neighborhood
+of each pixel. When using ``full_search=True`` the algorithm will return
+a single tree for the whole image provided that ``max_dist`` is large enough
+or that ``return_tree=True``.
+
+Since computing a full hierarchy requires searching the
+whole image it is also slower.
+
+
+Quickshift image segmentation
+-----------------------------
+
+Quickshift is a relatively recent 2D image segmentation algorithm, based on an
+approximation of kernelized mean-shift. Therefore it belongs to the family of
+local mode-seeking algorithms and is applied to the 5D space consisting of
+color information and image location [2]_.
+
+One of the benefits of quickshift is that it actually computes a
+hierarchical segmentation on multiple scales simultaneously.
+
+Quickshift has two main parameters: ``sigma`` controls the scale of the local
```

Comment:
It is called `kernel_size` in the `quickshift` function

Reply:
I am ok to change it but just so you know this is a copy paste from [the generic segmentation example](https://github.com/scikit-image/scikit-image/blob/d8f23c64c45427962ec5e7f975140de2d726547d/doc/examples/segmentation/plot_segmentations.py#L37). Do you want me to update it as well?